recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
     - pip
     - setuptools
     - wheel
+    - cython >=0.29.24
   run:
     - python
 
```

Comment:
It would be nice to add `httptools.parser` to `test/imports`, see https://github.com/MagicStack/httptools/tree/v0.6.4/httptools